packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/detachLine/detach-line.html

```diff
@@ -0,0 +1,82 @@
+<div class="telecom-telephony-line-phone-detach">
+    <header>
+        <oui-back-button
+            data-on-click="::$ctrl.goBack()"
+            previous-page="{{::'telephony_group_line_phone_section_back_link_target' | translate}}"
+        >
+        </oui-back-button>
+    </header>
+    <tuc-toast-message></tuc-toast-message>
+
+    <!-- DETACH -->
+    <section class="col-xs-12 col-md-6 widget-presentation">
```

Comment:
**question:** In bootstrap 4, the class name is `col-12`. Could you please check this behaviour?

Reply:
I fixed with only "col-12" I have same result.